src/elm/PageState/Auth/Main.elm

```diff
@@ -511,7 +514,7 @@ viewNavBar model =
                 [ i [ class "icon-plus" ] []
                 , text " New Password"
                 ]
-            , button [ class "newMasterKey btn", onClick OpenNewMasterKeyModal ]
+            , button [ class "newMasterKey btn", disabled (not model.config.masterKeyAllowEdit), onClick OpenNewMasterKeyModal ]
```

Comment:
De check bij update hoeft ook niet :p
De disabled is goed genoeg. 
De check bij update had ik toegevoegd als vangnet. Voor geval we nog een masterkey button zouden toevoegen en de disabled zouden vergeten. Die check kan dus sowieso weg